ui/client/components/ViewFeatures.js

```diff
@@ -353,7 +335,7 @@ const ViewFeatures = withStyles((theme) => ({
         }}
         loading={featuresLoading}
         getRowId={(row) => `${row.owner_dataset.id}-${row.name}`}
-        columns={displayableColumns}
+        columns={featureColumns}
```

Comment:
Displaying `Match %` column all the time, since it applies to any search (hybrid, the only one available now). Decided not to hide column when listing them.